psalm.xml

```diff
@@ -30,7 +30,7 @@
         <DeprecatedInterface>
             <errorLevel type="info">
                 <!-- Will be removed in 6.0.0 issues/211 -->
-                <referencedClass name="phpDocumentor\Reflection\DocBlock\Tags\Factory\StaticMethod"/>
+                <referencedClass name="phpDocumentor/Reflection/DocBlock/Tags/Factory/StaticMethod"/>
```

Comment:
i think you need `\\`?

Reply:
No, I think this was a wrong fix. I make the "windows directory separator" a lot in psalm, but this is a class name, not a path. I believe Psalm was screaming because referencedClass  was introduced in a newer version. (IIRC, the version used in CI is 3.0.X)

Reply:
Let me check, I found a way to install a newer psalm version

Reply:
> Let me check, I found a way to install a newer psalm version

Nice!